src/services/pure-di.tsx

```diff
@@ -118,7 +117,15 @@ export class AppCompositionRoot implements AppServices {
   }
 
   get shopifyClient() {
-    return this.singleton('shopifyClient', () => new HardcodedShopifyClient());
+    return this.singleton(
+      'shopifyClient',
+      () =>
+        new HttpShopifyClient({
+          axiosStatic: this.axiosStatic,
+          baseUrl: 'https://sfy.fromdoppler.com:4433',
```

Comment:
Remember to move it to the configuration file.

Also, I have already merge it to production, you can remove the port.

Reply:
I've added configurations but prefer to leave it hardcoded for testing purposes.